core/full_menu_view.js

```diff
@@ -0,0 +1,522 @@
+/* jslint node: true */
+'use strict';
+
+//  ENiGMA½
+const MenuView = require('./menu_view.js').MenuView;
+const ansi = require('./ansi_term.js');
+const strUtil = require('./string_util.js');
+const formatString = require('./string_format');
+const pipeToAnsi = require('./color_codes.js').pipeToAnsi;
+
+//  deps
+const util = require('util');
+const _ = require('lodash');
+
+exports.FullMenuView = FullMenuView;
+
+function FullMenuView(options) {
+  options.cursor = options.cursor || 'hide';
+  options.justify = options.justify || 'left';
+
+
+  MenuView.call(this, options);
+
+
+  // Initialize paging
+  this.pages = [];
+  this.currentPage = 0;
+
+  this.initDefaultWidth();
+
+
+
+  const self = this;
```

Comment:
For newer Enig code you can use ES6. This means you can use arrow functions (`=>`) that respect what you're used to for `this`... so no need to capture a `self`, either.

Reply:
I believe resolved. I didn't go all the way to convert to an ES6 class (might be cool to think about in the future, but would impact the full object hierarchy) but let me know if what I changed is good for now.

Reply:
Looks great! Some of it is a bit tricky due to the code base. When I started, ES6 wasn't quite ready, so you can see this evolution in the code. You'll see this around arrow functions as well. Async/await is not available also and I'd like to start producing more APIs to support it to make the code cleaner, but that's how laundry lists just keep getting longer :D